Notebooks/transcript_dataframe_builder.ipynb

```diff
@@ -0,0 +1,1264 @@
+       "                                              sentences annotations  index  \\\n",
+       "213   {'index': 213, 'confidence': 0.97, 'start_time...         NaN    213   \n",
+       "507   {'index': 507, 'confidence': 0.97, 'start_time...         NaN    507   \n",
+       "132   {'index': 132, 'confidence': 0.97, 'start_time...         NaN    132   \n",
+       "1171  {'index': 1171, 'confidence': 0.97, 'start_tim...         NaN   1171   \n",
+       "1426  {'index': 1426, 'confidence': 0.97, 'start_tim...         NaN   1426   \n",
+       "\n",
+       "      confidence  start_time   end_time  \\\n",
+       "213         0.97    1534.933   1548.580   \n",
+       "507         0.97    3532.795   3533.396   \n",
+       "132         0.97     949.448    953.319   \n",
+       "1171        0.97   10640.296  10643.199   \n",
+       "1426        0.97    9846.736   9860.517   \n",
+       "\n",
+       "                                                  words  \\\n",
+       "213   [{'index': 0, 'start_time': 1544.109, 'end_tim...   \n",
+       "507   [{'index': 0, 'start_time': 3532.795, 'end_tim...   \n",
+       "132   [{'index': 0, 'start_time': 949.448, 'end_time...   \n",
+       "1171  [{'index': 0, 'start_time': 10641.564, 'end_ti...   \n",
+       "1426  [{'index': 0, 'start_time': 9858.515, 'end_tim...   \n",
+       "\n",
+       "                                                   text speaker_index  \\\n",
+       "213   I'm just wondering--I know we had a really rob...             7   \n",
+       "507                                            They do?           119   \n",
+       "132                     It has been moved and Seconded.            44   \n",
+       "1171  We spend all day with these kids who look up t...           248   \n",
+       "1426  Sdot 3 B 1 sponsored by Council member Strauss...           174   \n",
+       "\n",
+       "     speaker_name annotations  \n",
+       "213          None        None  \n",
+       "507          None        None  \n",
+       "132          None        None  \n",
+       "1171         None        None  \n",
+       "1426         None        None  "
+      ]
+     },
+     "execution_count": 50,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "transcript_df.sample(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "\n",
+    "<center><h2>Exporting DataFrame<center><h2>\n",
+    "    \n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* Now that we have created a dataframe full of all available transcripts for our council, we can export this dataframe for analysis at a later date.\n",
+    "* In this example we will use .csv format."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 51,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transcript_df.to_csv('seattle-transcripts.csv', index = False)"
```

Comment:
Because the dataframes are storing a column of `Dict`, I might recommend storing with parquet instead of CSV. Should be faster and lead to less errors?

Reply:
I am googling parquet as we speak...